src/javascript/CopyToOtherLanguages/CopyAllToOtherLanguagesActionComponent.jsx

```diff
@@ -0,0 +1,69 @@
+import React, {useContext} from 'react';
+import PropTypes from 'prop-types';
+import {useTranslation} from 'react-i18next';
+import {ComponentRendererContext} from '@jahia/ui-extender';
+import {CopyToOtherLanguages} from './CopyToOtherLanguages';
+import {useContentEditorContext} from '@jahia/jcontent';
+import {useNodeChecks} from '@jahia/data-helper';
+import {useFormikContext} from 'formik';
+
+export const CopyAllToOtherLanguagesActionComponent = ({
```

Comment:
I feel existing action component could also have been reused here with a boolean param to differentiate during register, but I'm also ok to have them separate.